inspirehep/utils/tickets.py

```diff
@@ -24,26 +24,222 @@
 
 from __future__ import absolute_import, division, print_function
 
-from flask import current_app
-from rt import Rt, ConnectionError
+from os.path import join
+
+from flask import render_template
+
+from rt import ALL_QUEUES, ConnectionError
+
+from .proxies import rt_instance
 
 
 def get_instance():
```

Comment:
this is not the break the existing code, after refactoring the code base, I will remove.